lib/vmdb/gettext/debug.rb

```diff
@@ -0,0 +1,31 @@
+require 'fast_gettext'
+
+# include this module to see translations in the UI
+module Vmdb
+  module Gettext
+    module Debug
+      DL = "\u00BB".encode("UTF-8")
+      DR = "\u00AB".encode("UTF-8")
+
+      # slightly modified copy of fast_gettext _ method
+      def _(key)
+        DL + FastGettext._(key) + DR
```

Comment:
On a second read, this can also be written simpler. Since you are including the module, you can leverage `super`

```ruby
"#{DL}#{super}#{DR}"
```